.github/workflows/plugin-ci.yml

```
@@ -0,0 +1,114 @@
+
+jobs:
+  check:
+    name: Check
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+
+    steps:
+      - name: Check out MindRoom
+        uses: actions/checkout@df4cb1c069e1874edd31b4311f1884172cec0e10 # v6.0.3
+        with:
+          repository: ${{ inputs.mindroom_repository || job.workflow_repository }}
```

actionlint (1.7.12)

[error] property "workflow_repository" is not defined in object type {check_run_id: number; container: {id: string; network: string}; services: {string => {id: string; network: string; ports: {string => string}}}; status: string}

(expression)